nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc

```diff
@@ -10,7 +10,30 @@ The {secondary-scheduler-operator-full} allows you to deploy a custom secondary
 
 These release notes track the development of the {secondary-scheduler-operator-full}.
 
-:operator-name: The {secondary-scheduler-operator}
-include::snippets/operator-not-available.adoc[]
-
 For more information, see xref:../../../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[About the {secondary-scheduler-operator}].
+
+[id="secondary-scheduler-operator-release-notes-1.2.0"]
+== Release notes for {secondary-scheduler-operator-full} 1.2.0
+
+Issued: 2023-11-01
+
+The following advisory is available for the {secondary-scheduler-operator-full} 1.2.0:
+
+* link:https://access.redhat.com/errata/RHSA-2023:6154[RHSA-2023:6154]
```

Comment:
This link returns a 404 error. Probably because the target isn't included in the preview, but I wanted to  mark it just in case.

Reply:
Hi @nalhadef thanks for checking. It's expected that the advisory link will 404 until the release date tomorrow (I have a note in the description about it, but I know that's easy to miss). Thanks!